app/workers/github/create_issue_job.rb

```diff
@@ -16,7 +16,9 @@ class CreateIssueJob
 
     # :nocov:
     def perform(feedback)
-      Github::GithubService.create_issue(feedback)
+      feedback = Feedback.new(feedback)
+      create_response = Github::GithubService.create_issue(feedback)
+      FeedbackSubmissionMailer.build(feedback, create_response&.url).deliver_now
```

Comment:
`create_response&.url` will raise error if create_issue returns false

Reply:
Good catch.  Inside `GithubService#create_issue` I will go back to returning `nil` if an exception is thrown (and caught), e.g.
```ruby
rescue => e
  log_exception_to_sentry(e)
  # false
end
```

that way, `nil&.url` will return `nil`.